.travis.yml

```diff
@@ -16,13 +15,12 @@ notifications:
   email: false
 
 after_success:
-  - julia -e 'include(Pkg.dir("DocStringExtensions", "test", "coverage.jl"))'
+  - julia test/coverage.jl
 
 jobs:
   include:
     - stage: "Deploy docs"
-      julia: 0.6
+      julia: 1.0
       os: linux
-      script: julia -e 'Pkg.add("Documenter"); Pkg.checkout("Documenter")'
-        && julia ${TRAVIS_BUILD_DIR}/docs/make.jl
+      script: julia -e 'using Pkg; Pkg.add("Documenter"); include("docs/make.jl")'
```

Comment:
Soooo I think this will error without a project file, since `DocStringExtension` itself is not installed.

Reply:
Add a `Pkg.add(Pkg.PackageSpec(path=pwd()))`?

Reply:
(Shhh, I'm trying to get permission to include the project file :wink: )